spec.bs

```diff
@@ -763,70 +763,258 @@ enrollment and authentication. This extends the
+credential creation, attempting to trick the user into thinking that the
+WebAuthn credential creation is for logging into the legitimate parent site
+(`https://example.org`). The user creates the credential. The malicious actor
+can then later utilize `get()` in a cross-origin iframe (again, timing the call
+to coincide with a login attempt to `https://example.org`) to have the user
+accidentally identify themselves to the tracker iframe.
+
+This attack, however, presumes that `https://website.tracker` already has
+script access to `https://example.org` (in order to create the attacker iframe,
+with the appropriate permissions set). If this is true, then
+`https://website.tracker` can already directly track the user, for example by:
+
+* Identifying them from the username they provide when logging in to
+    `https://example.org`, OR
+* Creating a WebAuthn credential for `https://example.org` (in the main frame),
+    saving that to a `https://website.tracker` server, and later using that
+    credential (again in the main frame) to identify and track the user.
 
-* Credential ID(s) as a tracking vector
+<div class="note">
+**TODO**: The described attack does allow `https://website.tracker` to track the
+          user across multiple parent sites that it is directly included in,
+          using the credential as a cross-site identifier. The above
+          counter-claims stand up less well there, so we should perhaps remove
+          them.
+</div>
 
-     * The credential ID could be used as a tracking vector, but to obtain it
-         from the Relying Party the merchant already needs an as-strong
-         identifier to give to the Relying Party (e.g., the credit card number).
+In the absence of script access to the main frame, the ability for a malicious
+iframe to perform an attack like this seems far less feasible.  Firstly, the
+main frame must have granted the malicious iframe
+[[#sctn-permissions-policy|permission]] to call the API. Secondly, the iframe
+has no way to know when the user is logging into `https://example.org`, to
+align their attack. 
+
+A consideration for a future version of this specification might be to
+additionally require consuming a [=transient activation=] for credential
+creation in a cross-origin iframe, to ensure that the user has at least
+interacted with an iframe attempting credential creation.
+
+## Probing for credential ids ## {#sctn-privacy-probing-credential-ids}
+
+As per WebAuthn's section on [[webauthn-3#sctn-assertion-privacy|Authentication
+Ceremony Privacy]], implementors of Secure Payment Confirmation must make sure
+not to enable malicious callers (who now may not even be the [=Relying Party=]
+to distinguish between these cases:
+
+* A credential is not available.
+* A credential is available, but the user does not consent to use it.
+
+For example, the User Agent must fetch the {{PaymentCredentialInstrument/icon}}
+passed to SPC before checking whether any of the passed credentials match.
```

Comment:
Please rephrase this paragraph, to not start with "For example" and include a "must" requirement. (I see that the requirement is actually made in 4.1.4.) 

How about:

"This is why, in the Steps to check if a payment can be made, the User Agent fetches the {{PaymentCredentialInstrument/icon}} before checking whether any of the passed credential match."